test/xspec-tvt.xspec

```diff
@@ -0,0 +1,296 @@
+	-->
+
+	<x:scenario label="function-param">
+		<x:scenario label="user-content">
+			<x:call function="one-or-more">
+				<x:param>
+					<function-param-child x:expand-text="yes">}}{false()}{{</function-param-child>
+					<function-param-child expand-text="yes">}}{false()}{{</function-param-child>
+				</x:param>
+			</x:call>
+			<x:like label="user-content expect" />
+		</x:scenario>
+
+		<x:scenario label="@href">
+			<x:call function="exactly-one">
+				<x:param href="xspec-tvt.xml" />
+			</x:call>
+			<x:like label="href expect" />
+		</x:scenario>
+	</x:scenario>
+
+	<x:param name="global-param-user-content">
+		<global-param-child x:expand-text="yes">}}{false()}{{</global-param-child>
+		<global-param-child expand-text="yes">}}{false()}{{</global-param-child>
+	</x:param>
+	<x:param href="xspec-tvt.xml" name="global-param-href" />
+	<x:scenario label="global-param">
+		<x:scenario label="user-content">
+			<x:call function="one-or-more">
+				<x:param select="$global-param-user-content" />
+			</x:call>
+			<x:like label="user-content expect" />
+		</x:scenario>
+
+		<x:scenario label="@href">
+			<x:call function="exactly-one">
+				<x:param select="$global-param-href" />
+			</x:call>
+			<x:like label="href expect" />
+		</x:scenario>
+	</x:scenario>
+
+	<x:variable name="global-variable-user-content">
+		<global-variable-child x:expand-text="yes">}}{false()}{{</global-variable-child>
+		<global-variable-child expand-text="yes">}}{false()}{{</global-variable-child>
+	</x:variable>
+	<x:variable href="xspec-tvt.xml" name="global-variable-href" />
+	<x:scenario label="global variable">
+		<x:scenario label="user-content">
+			<x:call function="one-or-more">
+				<x:param select="$global-variable-user-content" />
+			</x:call>
+			<x:like label="user-content expect" />
+		</x:scenario>
+
+		<x:scenario label="@href">
+			<x:call function="exactly-one">
+				<x:param select="$global-variable-href" />
+			</x:call>
+			<x:like label="href expect" />
+		</x:scenario>
+	</x:scenario>
+
+	<x:scenario label="local variable">
+		<x:variable name="local-variable-user-content">
+			<local-variable-child x:expand-text="yes">}}{false()}{{</local-variable-child>
+			<local-variable-child expand-text="yes">}}{false()}{{</local-variable-child>
+		</x:variable>
+		<x:variable href="xspec-tvt.xml" name="local-variable-href" />
+		<x:scenario label="user-content">
+			<x:call function="one-or-more">
+				<x:param select="$local-variable-user-content" />
+			</x:call>
+			<x:like label="user-content expect" />
+		</x:scenario>
+
+		<x:scenario label="@href">
+			<x:call function="exactly-one">
+				<x:param select="$local-variable-href" />
+			</x:call>
+			<x:like label="href expect" />
+		</x:scenario>
+	</x:scenario>
+
+	<x:scenario label="assertion">
+		<x:scenario label="user-content">
+			<x:call function="false" />
+			<x:expect label="@x:expand-text=yes enables TVT and @x:expand-text is dicarded"
```

Comment:
Typo in the word discarded

Reply:
Fixed it in a74ce8cbbb1cdbb708e49b25bb6776af9383f3ef, and you yielded another [contribution](https://github.com/codespell-project/codespell/commit/26e6102b699acb160d50de32cc552dc7e22fb377)... 😀

Reply:
Cool! I didn't know about that spelling checker.

Reply:
codespell was integrated into our CI by #675, just FYI.